libs/platform/src/lib/form/multi-combobox/commons/base-multi-combobox.ts

```diff
@@ -367,6 +386,12 @@ export abstract class BaseMultiCombobox extends CollectionBaseInput implements A
         super(_cd, ngControl, ngForm, formField, formControl);
     }
 
+    ngOnChanges(changes: SimpleChanges): void {
+        if (changes['dataSource']) {
```

Comment:
well, you already have input setter on dataSource, you can use that and shrink lines by removing ngOnChanges

Reply:
Or you can remove dataSource setter, if always getting the filled input is not something very important and shrink code that way

Reply:
Either way, in this case, one of them should go in my opinion